### PR TITLE
Remove Node 12 from CI/CD

### PR DESCRIPTION
**Pull request by coliff**
Removes Node 12 from CI/CD

Fixes: #1115

### CODE REVIEW

File: .github/workflows/development.yml [hunk 1]
```diff
@@ -66,23 +65,23 @@ jobs:
         run: git diff --exit-code
 
       - name: Run tests
-        if: matrix.node != '12' || matrix.os != 'ubuntu-latest'
+        if: matrix.os != 'ubuntu-latest'
```
Comment (nschonni) [inline .github/workflows/development.yml:68]: This needs to be pointed at a different version, as it's used to make sure certain tasks are only run once

Comment (coliff) [reply .github/workflows/development.yml:68]: Ah, ok, thanks. I’ll change it to 14 then.

File: .github/workflows/development.yml [hunk 2]
```diff
@@ -66,23 +65,23 @@ jobs:
...
+        if: matrix.node != '14' || matrix.os != 'ubuntu-latest'
         uses: nick-invision/retry@v2
         with:
           timeout_minutes: 20
           max_attempts: 3
           command: npm run test
 
       - name: Run coverage
-        if: matrix.node == '12' && matrix.os == 'ubuntu-latest'
+        if: matrix.node != '14' || matrix.os == 'ubuntu-latest'
```
Comment (nschonni) [inline .github/workflows/development.yml:76]: You changed the meaning here by going from AND to OR

Comment (nschonni) [reply .github/workflows/development.yml:76]: Might be better to use 16, since that is the current LTS release and won't expire for longer

**Review by nschonni — APPROVED** [review]

**Pull request merged**